modules/optics/arrow-optics/src/main/kotlin/arrow/optics/Optional.kt

```diff
@@ -79,15 +79,6 @@ interface POptional<S, T, A, B> : POptionalOf<S, T, A, B> {
       override fun set(s: S, b: B): T = set(s, b)
     }
 
-    /**
-     * Invoke operator overload to create a [POptional] of type `S` with focus `A`.
-     * Can also be used to construct [Optional]
-     */
-    operator fun <S, A> invoke(partialFunction: PartialFunction<S, A>, set: (S, A) -> S): Optional<S, A> = Optional(
```

Comment:
Don't we need a version with `(S) -> Option<A>` to replace the `PartialFunction<S, A>`?